### PR TITLE
Set-up dotenv

### PR DESCRIPTION
**Pull request by e-xe**
I've done a small setup of `dotenv`, now, users can create a `.env` file in the workdir and the code will look like this, eg: Running the bot with a token (gotta keep that private)
```py
import discord
from utils import config

dotenv_vars = config.envs

bot = discord.ext.commands.Bot(command_prefix='!')

bot.run(dotenv_vars('BOT_TOKEN'))
```
Explanation: Config.py has a script where it runs `dotenv.load_env()`, that will add environment variables from the `.env` file on the system's environment variable and `config.envs` is an instance of `os.getenv`.

There is further setup to do with it which I will do later but for now, the bot is in a condition to run.

### CODE REVIEW

File: src/__main__.py [hunk 1]
```diff
@@ -1,44 +1,27 @@
...
-
-async def change_status() -> None:
-    await client.wait_until_ready()
-    msgs = cycle(status)
-
-    while not client.is_closed():
-        current_status = next(msgs)
-        await client.change_presence(activity=discord.Game(name=current_status))
-        await asyncio.sleep(10800)
-
```
Comment (janaSunrise) [inline src/__main__.py:36]: **DON'T REMOVE THIS PART!**

File: src/__main__.py [hunk 2]
```diff
@@ -1,44 +1,27 @@
...
 
 @client.event
 async def on_ready() -> None:
     print("Bot is Ready.")
     print(f"Logged in as: {client.user.name} : {client.user.id}")
+    await client.change_presence(activity=discord.Activity(type=discord.ActivityType.watching, name="for stranded peeps"))
```
Comment (janaSunrise) [inline src/__main__.py:24]:
```suggestion
    
```

File: src/__main__.py [hunk 3]
```diff
@@ -107,7 +90,6 @@ def setup_bot(bot: Bot) -> None:
     bot.run(TOKEN)
 
 
-client.loop.create_task(change_status())
```
Comment (janaSunrise) [inline src/__main__.py:110]: **DON'T DELETE THIS TOO!**

File: src/__main__.py [hunk 4]
```diff
@@ -1,44 +1,27 @@
 import asyncio
 import os
 import textwrap
-from itertools import cycle
```
Comment (janaSunrise) [inline src/__main__.py:4]: **REVERT DELETING THIS!**

File: src/__main__.py (hunk 1 above)
Comment (Theodore-Robinson) [reply src/__main__.py:36]: That is not a good status
We need to keep it clean and small

File: src/__main__.py (hunk 4 above)
Comment (janaSunrise) [reply src/__main__.py:4]:
```python
status = [
    "😁Working At The Codin' Hole! Join me at https://discord.gg/aYF76yY",
    "▶Check out My Creator's Youtube channel : https://www.youtube.com/channel/UC3S4lcSvaSIiT3uSRSi7uCQ/",
    f"Ping me using {PREFIX}help",
    "Watching {num of members} over {num server} across {num of shards} shards",
    "Ready To Work and Get Worked! My Github 🔆 https://github.com/janaSunrise",
]```
the status must be like this

File: .travis.yml [hunk 5]
```diff
@@ -15,4 +15,5 @@ install:
     - "pipenv sync -d"
 
 script:
+    - "pipenv run autoformat"
```
Comment (ItsDrike) [inline .travis.yml:18]: Why should Travis-CI run autoformat, he can't push those changes to GitHub anyway, that's the point of Travis-CI, it should only check things, not actually change them.  This should be removed.
```suggestion
```

File: Pipfile [hunk 6]
```diff
@@ -26,7 +26,7 @@ python_version = "3.8"
 [scripts]
 lint = "pre-commit run --all-files"
 precommit = "pre-commit install"
-autoformat = "black ."
+autoformat = "black src"
```
Comment (ItsDrike) [inline Pipfile:29]: Not sure this is a good idea, we might have some python files outside `src/` which also needs to be formatted with `black`

File: src/cogs/coding.py [hunk 7]
```diff
@@ -17,10 +17,10 @@
 from discord.ext.commands import Cog, Context
 from discord.utils import escape_mentions
 
-from .code_utils import _doc, _ref
-from .code_utils._used import get_raw, paste, typing
+from utils import _doc, _ref
+from utils._used import get_raw, paste, typing
 import typing as t
-from .code_utils._tio import Tio
+from utils._tio import Tio
```
Comment (ItsDrike) [inline src/cogs/coding.py:23]: All inputs should be sorted. VSCode can do this simply by using `Sort Imports` in Command Pallete (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>)

File: src/__main__.py [hunk 8]
```diff
@@ -1,44 +1,27 @@
...
 
 import discord
 from discord.ext import commands
 from discord.ext.commands import Bot
 
-from cogs.utils import constants
+from utils import config
+envs = config.envs
```
Comment (ItsDrike) [inline src/__main__.py:10]:
```suggestion
from utils import config


envs = config.envs
```
`flake8` standards requite 2 lines gap between imports and the rest of the code

File: src/utils/config.py [hunk 9]
```diff
@@ -1,6 +1,11 @@
 from enum import Enum
-
+from dotenv import load_dotenv
 from discord import Color
+import os
```
Comment (ItsDrike) [inline src/utils/config.py:4]: These imports need to be sorted (I've already explained why in some review above)
They should look like this:
```py
import os
from enum import Enum

from discord import Color

from dotenv import load_dotenv
```

**Comment by ItsDrike** [comment]
> File structure change isn't needed now.

I disagree @janaSunrise we need to change the file structure to make the code more readable and overall better for future contributions

**Comment by janaSunrise** [comment]
> > File structure change isn't needed now.
> 
> I disagree @janaSunrise we need to change the file structure to make the code more readable and overall better for future contributions

But, we can create a PR for that

**Comment by ItsDrike** [comment]
> > > File structure change isn't needed now.
> > 
> > 
> > I disagree @janaSunrise we need to change the file structure to make the code more readable and overall better for future contributions
> 
> But, we can create a PR for that

Literally this is the PR for that, lol.

**Comment by janaSunrise** [comment]
> > > > File structure change isn't needed now.
> > > 
> > > 
> > > I disagree @janaSunrise we need to change the file structure to make the code more readable and overall better for future contributions
> > 
> > 
> > But, we can create a PR for that
> 
> Literally this is the PR for that, lol.

then remove the status changes and env updates, I did it, will update later

**Comment by ItsDrike** [comment]
> then remove the status changes and env updates, I did it, will update later

No, they can be implemented here we don't need to split this PR when it's already made.

**Pull request closed without merging**